src/react/components/VersionFooter.tsx

```diff
@@ -31,9 +31,9 @@ const VersionFooter = () => {
         <Grid item xs={2} />
         <Grid item xs={8}>
           <SoftText>
-            GUI: {VERSION}#{COMMITHASH}
+            GUI: {VERSION}
             <br/>
-            Deposit-CLI: {CLIVERSION}#{CLICOMMITHASH}
```

Comment:
here you can keep the both commit hashes added to the version actually. same for the GUI version

Reply:
I reverted the change although I added parenthesis around the hash. That way it will say `Dposit-CLI: 1.x (#HASH)` making it more obvious that the version number and the commit hash are not part of the same string.

Reply:
Added links to each of the commit hashes to go to the github URL for that tree.